### PR TITLE
Update `create_instance` workflow

### PR DESCRIPTION
**Pull request by vutny**
### What's this PR do?
Adds support for `BlockDeviceMappings` and `IamInstanceProfile` parameters in `create_instance` example workflow.

@jjm @Kami @LindsayHill Please review. Thanks!

### CODE REVIEW

**Comment by Kami** [comment]
Thanks for the contribution.

The change looks good to me, but I haven't had the chance to test it.

File: actions/create_instance.yaml [hunk 1]
```diff
@@ -30,6 +30,16 @@ parameters:
     type: "string"
     description: "SSH key to use during intial instance creation"
     required: false
+  BlockDeviceMappings:
```
Comment (Kami) [inline actions/create_instance.yaml:33]: Do we need to default this to an empty array, same as we do for ``SecurityGroupIds`` so the workflow won't fail?

File: actions/create_instance.yaml [hunk 2]
```diff
@@ -30,6 +30,16 @@ parameters:
...
+    type: "array"
+    items:
+      type: "object"
+    description: "The block device mapping entries"
+    required: false
+  IamInstanceProfile:
+    type: "object"
```
Comment (Kami) [inline actions/create_instance.yaml:40]: Same here - probably also need to default it to an empty object so it works if this parameter is not provided.

Would be good to test those scenarios as well (making sure it works correctly when those two parameters are not provided).

**Comment by vutny** [comment]
Thanks for your review @Kami !
Indeed, the default values are needed, otherwise they may not be rendered correctly in the workflow.
I will make changes and do tests, then get back here.

**Comment by vutny** [comment]
@Kami Works great after setting the defaults.
```
st2 run --auto-dict aws_boto3.create_instance ImageId=ami-9887c6e7 Tags=Name:my-instance Tags=Owner:me@example.com
..................
id: 5cdd66f216da0b042951524c
action.ref: aws_boto3.create_instance
parameters: 
  ImageId: ami-9887c6e7
  Tags:
  - Name: my-instance
  - Owner: me@example.com
status: succeeded
result_task: add_tags
result: 
  exit_code: 0
  result:
    ResponseMetadata:
      HTTPHeaders:
        content-length: '221'
        content-type: text/xml;charset=UTF-8
        date: Thu, 16 May 2019 13:35:18 GMT
        server: AmazonEC2
      HTTPStatusCode: 200
      RequestId: 5f542d0d-20d7-489d-8d94-b71d7b515b44
      RetryAttempts: 0
  stderr: ''
  stdout: ''
start_timestamp: Thu, 16 May 2019 13:34:42 UTC
end_timestamp: Thu, 16 May 2019 13:35:19 UTC
+--------------------------+-------------------------+-------------------+-----------------------+-------------------------------+
| id                       | status                  | task              | action                | start_timestamp               |
+--------------------------+-------------------------+-------------------+-----------------------+-------------------------------+
| 5cdd66f316da0b007233fe0b | succeeded (2s elapsed)  | run_instance      | aws_boto3.boto3action | Thu, 16 May 2019 13:34:43 UTC |
| 5cdd66f516da0b007233fe0d | succeeded (31s elapsed) | wait_for_instance | aws_boto3.waiter      | Thu, 16 May 2019 13:34:45 UTC |
| 5cdd671516da0b007233fe0f | succeeded (1s elapsed)  | add_tags          | aws_boto3.boto3action | Thu, 16 May 2019 13:35:17 UTC |
+--------------------------+-------------------------+-------------------+-----------------------+-------------------------------+
```

**Comment by LindsayHill** [comment]
Sorry, this should have been merged a while ago!

**Comment by vutny** [comment]
Thanks a lot!

**Pull request merged**